Gemfile

```diff
@@ -8,10 +8,15 @@ group :development, :test do
   gem 'pry' unless ENV['CI']
   gem 'pry-byebug' unless ENV['CI']
   gem "simplecov", require: false
+  # Override the coffee-rails version in engine_cart for rails 4.2
+  case ENV['RAILS_VERSION']
+  when /^4.2/
+    gem 'coffee-rails', '~> 4.2.0'
```

Comment:
The line-44 code is maintained by engine-cart, it should not be modified by anything but engine cart and we should assume that engine-cart will not respect any changes in the block it maintains.  To override anything in that block, it must be declared separately.  So, in this PR, to preserve the intention of a prior commit, this block was added in the last commit.